magpie-aws/src/test/java/io/openraven/magpie/plugins/aws/discovery/services/IAMAccountDiscoveryIT.java

```diff
@@ -71,7 +71,6 @@ private void assertAccount(MagpieEnvelope envelope) {
     assertEquals(alias, contents.get("resourceName").asText());
     assertEquals("AWS::Account", contents.get("resourceType").asText());
     assertEquals(ACCOUNT, contents.get("awsAccountId").asText());
-    assertEquals(BASE_REGION.toString(), contents.get("awsRegion").asText());
```

Comment:
When we spoke about this last week, we opted to add:

        .withAwsRegion(region.id())
at IAMDiscovery.jave:392.

Change of heart?

Reply:
Was that added and I missed it?  If not, my thought process was that I'd rather change the test rather than change discovery as I feared unintended consequences downstream.